### DISCUSSION TITLE
Question about YAML outputting

### DISCUSSION THREAD

**Discussion by Mar0xy** [WeblateOrg/weblate#10394] [Q&A]
I am wondering why strings in YAML files aren't outputted with quotes(") as it normally is sane to quote strings to prevent them from being recognized as booleans in case of no, false and etc or in case of numbers being turned into nums

**Comment by nijel** [comment]
The quotes are used only when needed (what is typical in YAML).

**Reply by Mar0xy** [reply]
That is nice to know just the issue is if the locale file prior used to always use quotes it would then proceed to look messy as the lines weblate then proceeds to change will have the quotes removed which is I think adding an option to also just always include quotes in the Formatting addon would be an appreciate feature especially for people that always use quotes in their locales.

**Reply by nijel** [reply]
The quoting of existing strings is preserved (just to verify this, I've added tests in https://github.com/translate/translate/pull/5126). Quoting of newly added strings currently cannot be configured.

**Reply by Mar0xy** [reply]
Well that is sad, Guess we need to look into Crowdin as an alternative as they quote new strings in YAML files :disappointed: 
but thanks for answering.

**Reply by nijel** [reply]
If there is a way to configure this in ruamel.yaml, it would be easy to add to [YAML customization add-on](https://github.com/WeblateOrg/weblate/blob/main/weblate/addons/yaml.py).
